doc/content/integrations/lora-cloud/das.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "LoRa Cloud Device & Application Services"
+description: ""
+---
+
+[LoRa Cloud™ Device & Application Services](https://www.loracloud.com/portal/device_management/home) comprise a set of full life-cycle management features for LoRaWAN®-based devices, which are intended to simplify the development of managed LoRaWAN endpoint solutions.
```

Comment:
We don't have to put (R) on LoRaWAN in documentation content. We usually do that once, on the landing page or something, but we don't have to bother with documentation content

Reply:
Same for LoRa Edge BTW, only (TM) the first use